src/types/observableobject.ts

```diff
@@ -87,7 +87,7 @@ export class ObservableObjectAdministration
         public name: string,
         public defaultEnhancer: IEnhancer<any>
     ) {
-        this.keysAtom = new Atom(name + ".keys")
+        this.keysAtom = new Atom(name + ".keys") as IAtom
```

Comment:
Perhaps it might be better to set up an actual property `private keysAtom: IAtom` so it's not necessary to force type here.

Reply:
@JSlife23 Bump? :)

Reply:
It is already declared as IAtom, so cast shouldn't be needed.